pkg/provisioner/ironic/validatemanagementaccess_test.go

```diff
@@ -136,6 +136,71 @@ func TestValidateManagementAccessExistingNode(t *testing.T) {
 	assert.Equal(t, "uuid", host.Status.Provisioning.ID)
 }
 
+func TestValidateManagementAccessExistingNodeStatus(t *testing.T) {
+	statuses := []nodes.ProvisionState{
```

Comment:
`Verifying` isn't in this list, and it's going to have a different expected behavior. I don't see another test that covers that case. How many other states are missing here?

Reply:
This is testing all the ones from Manageable on to verify that they don't return Dirty. So only Enroll and Verifying are missing.

Reply:
Added tests for the other 2 states.